### PR TITLE
Added the paradox documentation

### PR DESCRIPTION
**Pull request by bogdanromanx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,67 +33,67 @@ Now let's take a look at these components individually.
 ### Nexus KnowledgeGraph
 This service is the heart of the Blue Brain Nexus platform. It allows users to define their domain, populate the knowledge graph with data, attach data files to entities. It also provides semantic search facilities to discover similar and relevant data in the platform.
 
-**Status**: active development
-**Source code**: https://github.com/BlueBrain/nexus-kg
+**Status**: active development  
+**Source code**: https://github.com/BlueBrain/nexus-kg
```
Comment (wwajerowicz) [inline README.md]: Is there a reason this whitespace was added here?

Comment (bogdanromanx) [reply README.md]: It was intended, to get a line break.

**Comment by bogdanromanx** [comment]
Good catch, however I wouldn't add build instructions to the project readme... I would very much prefer to fix the issue (by publishing the artifacts to a public repo).

File: src/main/paradox/setup/resources/index-domains.properties [hunk 2]
```diff
@@ -0,0 +1,11 @@
+com.bigdata.rdf.sail.namespace=kg-domains
+com.bigdata.rdf.sail.isolatableIndices=false
+com.bigdata.rdf.store.AbstractTripleStore.textIndex=true
+com.bigdata.rdf.store.AbstractTripleStore.quads=true
+com.bigdata.rdf.sail.truthMaintenance=false
+com.bigdata.namespace.test-ns.lex.com.bigdata.btree.BTree.branchingFactor=400
+com.bigdata.rdf.store.AbstractTripleStore.justify=false
+com.bigdata.rdf.store.AbstractTripleStore.geoSpatial=false
+com.bigdata.namespace.test-ns.spo.com.bigdata.btree.BTree.branchingFactor=1024
```
Comment (samuel-kerrien) [inline src/main/paradox/setup/resources/index-domains.properties]: test-ns ? is that something we want to put out int he default config ?

File: src/main/paradox/setup/resources/index-domains.properties [hunk 3]
```diff
@@ -0,0 +1,11 @@
+com.bigdata.rdf.sail.namespace=kg-domains
+com.bigdata.rdf.sail.isolatableIndices=false
+com.bigdata.rdf.store.AbstractTripleStore.textIndex=true
+com.bigdata.rdf.store.AbstractTripleStore.quads=true
+com.bigdata.rdf.sail.truthMaintenance=false
+com.bigdata.namespace.test-ns.lex.com.bigdata.btree.BTree.branchingFactor=400
```
Comment (samuel-kerrien) [inline src/main/paradox/setup/resources/index-domains.properties]: test-ns ?

File: src/main/paradox/setup/resources/index-domains.properties (hunk 2 above)
Comment (MFSY) [reply src/main/paradox/setup/resources/index-domains.properties]: For now, I guess these namespace configs need to be (manually) applied  prior to start Nexus KG service in order to benefit from indexing and search through blazegraph. 

But the right approach would be to read and applied (user provided or default) namespace configs automatically at boot time.

Comment (bogdanromanx) [reply src/main/paradox/setup/resources/index-domains.properties]: Very nice catch; I've just used the index definitions that were present in the docs. I'll regenerate the indices.

Comment (bogdanromanx) [reply src/main/paradox/setup/resources/index-domains.properties]: Corrected the issue.

File: src/main/paradox/setup/resources/index-domains.properties (hunk 3 above)
Comment (bogdanromanx) [reply src/main/paradox/setup/resources/index-domains.properties]: Corrected the issue.

**Comment by bogdanromanx** [comment]
Is this GTM?

**Comment by wwajerowicz** [comment]
looks fine to me

**Pull request merged**